docs/design.md

```diff
@@ -1,24 +1,26 @@
 <!--
```

Comment:
This is a commented out block, what defines it is `<!--` and `-->`, adding `#` on the license lines is redundant.

Reply:
I guess it depends on what the desire is. The HTML-style comments with <!-- and --> do copy the block into the final html, but it will be ignored by the browser. I'm not sure what adding '#' lines would mean.

Markdown doesn't really have a comment indicator, but it can be faked with something like:

    [comment]: # This is a comment

but that needs to be prefixed on every line (and is fairly ugly). I think we should leave it as it was.

[comment]: # Note that not all markdown parsers even handle these, and that github will show them.

Reply:
I modified it because more local markdown readers had issues with rendering the previous format. Now it works and looks fine (my markdown readers do show these comment, unlike GitHub for example).

Unfortunately, comments are not straightforward in Markdown.

Reply:
I had difficulties with the empty lines in the license text, that's why I had to use the '#' prefix for every line.